frontend/uddhar-frontend/src/shared/components/Navbar.jsx

```
@@ -10,15 +10,20 @@ import logo from "../../assets/uddhar.png";
     <>
       <nav className="top-0 left-0 right-0 w-full">
         <div>
           <div className="bg-amber-300 flex justify-between items-center px-15 h-17 max-md:px-7">
             <div>
-              <img src={logo} alt="Logo" className="min-w-30 min-h-30 w-30" />
+              <img
+                src={logo}
+                alt="Logo"
+                className="min-w-20 min-h-20 w-30 cursor-pointer"
+                onClick={() => (window.location.href = "/")}
+              />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)